apps/antalmanac/src/stores/calendarizeHelpers.ts

```diff
@@ -121,7 +123,7 @@ export function calendarizeFinals(currentCourses: ScheduleCourse[] = []): Course
 export function calendarizeCustomEvents(currentCustomEvents: RepeatingCustomEvent[] = []): CustomEvent[] {
     return currentCustomEvents.flatMap((customEvent) => {
         const dayIndiciesOcurring = customEvent.days.map((day, index) => (day ? index : undefined)).filter(notNull);
-
+        const days = dayIndiciesOcurring.map((dayIndex) => COURSE_WEEK_DAYS[dayIndex]);
```

Comment:
In the other helper functions, I included comments describing how these data structures look, e.g. here https://github.com/icssc/AntAlmanac/blob/85ff4a87e04507373e163ee65559caa67a8eaaaa/apps/antalmanac/src/stores/calendarizeHelpers.ts#L93

Personally, I think it's useful to see examples of the data being generated nearby the variable it's assigned to. I think it would be good to include some documentation here as well (you can copy/paste the previous comments too, since the data should be the same/similar). Let me know if you or @EricPedley have alternative opinions on this.

For example:
```ts
 /**
  * Only include the day indices that the final is occurring.
  *
  * @example [false, false, false, true, false, true, false] -> [3, 5]
  */
const dayIndicesOcurring = weekdaysOccurring.map((day, index) => (day ? index : undefined)).filter(notNull);
```

Reply:
Sounds solid! Yeah the comment about the data structure will be very helpful.